ansible/vars/layout/mainnet.yml

```
@@ -9,45 +9,44 @@ nodes_layout:
     - { branch: 'stable', history: 'archive' }
 
   # Innova Hosting ---------------------------------------------------
-  'linux-01.ih-eu-mda1.nimbus.mainnet': # Frankenstein host, all nodes use one Geth.
-    - { branch: 'stable',   num: 1 }
-    - { branch: 'testing',  num: 1, open_libp2p_ports: false }
-    - { branch: 'unstable', num: 1, public_api: true }
-    - { branch: 'unstable', num: 2 }
-
-  'linux-02.ih-eu-mda1.nimbus.mainnet':
-    - { branch: 'stable',   num: 1 }
-    - { branch: 'testing',  num: 1, public_api: true }
-    - { branch: 'unstable', num: 1, open_libp2p_ports: false }
-    - { branch: 'unstable', num: 2 }
-
-  'linux-03.ih-eu-mda1.nimbus.mainnet':
-    - { branch: 'stable',   num: 1, max_peers: 10000 }
-    - { branch: 'testing',  num: 1 }
-    - { branch: 'unstable', num: 1 }
-    - { branch: 'unstable', num: 2 }
-
-  'linux-04.ih-eu-mda1.nimbus.mainnet':
-    - { branch: 'stable',   num: 1 }
-    - { branch: 'testing',  num: 1 }
-    - { branch: 'unstable', num: 1 }
-    - { branch: 'unstable', num: 2 }
-
-  'linux-05.ih-eu-mda1.nimbus.mainnet':
-    - { branch: 'stable',   num: 1 }
-    - { branch: 'testing',  num: 1 }
-    - { branch: 'unstable', num: 1 }
-    - { branch: 'unstable', num: 2 }
-
-  'linux-06.ih-eu-mda1.nimbus.mainnet':
-    - { branch: 'stable',   num: 1 }
-    - { branch: 'testing',  num: 1 }
-    - { branch: 'unstable', num: 1 }
-    - { branch: 'unstable', num: 2 }
-
-  'linux-07.ih-eu-mda1.nimbus.mainnet':
-    - { branch: 'stable',   num: 1 }
-    - { branch: 'libp2p',   num: 1 }
-    - { branch: 'libp2p',   num: 2 }
-    - { branch: 'libp2p',   num: 3, extra_flags: {'debug-enable-yamux': true} }
-    - { branch: 'libp2p',   num: 4, extra_flags: {'debug-enable-yamux': true} }
+  'nel-01.ih-eu-mda1.nimbus.mainnet':
+    - { branch: 'stable',   el: 'nel' }
+    - { branch: 'testing',  open_libp2p_ports: false }
+    - { branch: 'unstable', el: 'nel', public_api: true }
+    - { branch: 'libp2p' }
```

I would make the alignment of `el` key such that if it's not set it's empty:
```suggestion
  'nel-01.ih-eu-mda1.nimbus.mainnet':
    - { branch: 'stable',   el: 'nel'                           }
    - { branch: 'testing',             open_libp2p_ports: false }
    - { branch: 'unstable', el: 'nel', public_api: true         }
    - { branch: 'libp2p'                                        }
```
For readability.